packages/schema/src/json-schema/shared.ts

```
@@ -0,0 +1,200 @@
+  const: true,
+  multipleOf: true,
+  maxProperties: true,
+  minProperties: true,
+  uniqueItems: true,
+};
+
+export const SchemaNormalKeys = Object.keys(SchemaStateMap);
+
+export const SchemaValidatorKeys = Object.keys(SchemaValidatorMap);
+
+export const hasOwnProperty = Object.prototype.hasOwnProperty;
```

Biome (1.9.4)

[error] Do not shadow the global "hasOwnProperty" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)